ui/packages/consul-ui/app/components/token-source/index.hbs

```diff
@@ -7,10 +7,10 @@ as |State Guard Action dispatch state|>
     @cond={{this.isSecret}}
   />
   {{#let
-    (uri '/${partition}/{$nspace}/${dc}'
+    (uri '/${partition}/${nspace}/${dc}'
       (hash
-        partition=@partition
-        nspace=@nspace
+        partition=(or @value.Partition @partition)
+        nspace=(or @value.Namespace @nspace)
```

Comment:
As mentioned above, now we pass the full OIDC provider object through we can access the Partition (and probably best to do the same with Namespace here so I added that also)